nucleus/modelci/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""Model CI Python Library."""
+
+__all__ = [
+    "ModelCI",
+    "UnitTest",
+]
```

Comment:
Curious why `UnitTestEvaluation` and `UnitTestItemEvaluation` aren't here.

Reply:
<img width="901" alt="Screen Shot 2021-12-08 at 10 39 18 PM" src="https://user-images.githubusercontent.com/70984140/145346686-67b42747-0b67-4dcc-96d9-1bcca0d87ce2.png">
My understanding is that it creates these shortcuts at the top of the page! 

We could add more classes later on but by default they're sorted alphabetically which is kind of weird (ModelCI should definitely be at the top).  It's difficult to figure out ordering so I'm inclined to leave that to a follow up PR

Reply:
We had more initially, but I think they just create confusion for now. We can also add them in, I just thought those two are the most important for interaction. Wdyt @phil-scale ?